1707237627375	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1707237925497	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1707237925498	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1707237925503	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=schedule)
1707237925505	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1707238071748	Sync.SyncScheduler	DEBUG	Woke from sleep.
1707238071749	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1707238071749	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1707238071751	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1707238073753	Sync.Service	DEBUG	User-Agent: Firefox/122.0 (Linux x86_64) FxSync/1.124.0.20240123151424.desktop
1707238073753	Sync.Service	INFO	Starting sync at 2024-02-06 19:47:53 in browser session pS7cALQHUw7u
1707238073754	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1707238073757	Sync.Status	INFO	Resetting Status.
1707238073757	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1707238073757	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1707238073757	FirefoxAccounts	DEBUG	getOAuthToken enter
1707238073758	FirefoxAccounts	TRACE	getCachedToken returning cached token
1707238073758	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1707238073758	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1707238073763	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1707238073763	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 479\"  data: no]"}
1707238073764	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/207020392/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1707238073764	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1707238073764	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1707238073764	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1707238073765	Sync.SyncScheduler	DEBUG	Next sync in 3600000 ms. (why=reschedule)
1707238073768	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
